### DISCUSSION TITLE
"need to implement importing another file"

### DISCUSSION THREAD

**Discussion by bcyong** [kobbled/tp_plus#31] [Q&A]
I received this error when playing with namespaces utilized across functions. Does this mean this feature still needs to be implemented in TP+, or that I need to do something in my code to enable this feature?

Relevant gist: https://gist.github.com/bcyong/41856aca83b1eabbaf111170b0867316
Lines 8-19 work fine, but 21-35 throw this error.

**Comment by kobbled** [comment]
Hi, can you post the full error message, I got it to interpret on my end. Make sure you run `rake` after a git pull to compile the parser. `parser.rb` is not tracked in the git, as depending on your dependency version (racc) you can get different results.  Also, is the gist what you are trying to run, or are you putting PRtypes in a separate file and trying to import it?

PRTypes.tpp
```
namespace PRTypes
  POSITION  := 1
  XYZWPR    := 2
  XYZWPREXT := 6
  JOINTPOS  := 9
end
```
main.tpp
```
import PRTypes
```

Like this ^.
